pkg/storaged/filesystem/utils.jsx

```diff
@@ -212,10 +216,8 @@ export const MountPoint = ({ fstab_config, forced_options, backing_block, conten
                 <FlexItem>{ mount_point_text }</FlexItem>
                 }
                 <FlexItem>
-                    <StorageLink onClick={() => mounting_dialog(client,
-                                                                content_block || backing_block,
-                                                                "update",
-                                                                forced_options, subvol)}>
+                    <StorageLink onClick={() => edit_mount_point(content_block || backing_block,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19973-20240208-140838-896b0145-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)